app/assets/stylesheets/campaigns/peer_to_peer/page.css.scss

```diff
@@ -11,6 +11,14 @@ body {
 	padding: 0;
 	background: $fog;
 }
+
+body > .campaign-banner {
```

Comment:
Is the campaign_banner always going to be 980px? Does it (or should it) vary/change size/etc on non-Desktop screens?

Reply:
For desktops and large screens it shouldn't be bigger, but yes: for mobile it should be smaller, accordingly to the whole layout.